client/state/utils/serialize.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { getInitialState } from '@automattic/state-utils';
+import type { Reducer, AnyAction } from 'redux';
+
+export type SerializableReducer< TState, TAction extends AnyAction > = Reducer<
+	TState,
+	TAction
+> & {
+	serialize?: ( state: TState ) => any;
+	deserialize?: ( persisted: any ) => TState;
+};
+
+export function serialize< TState, TAction extends AnyAction >(
```

Comment:
Does `TAction` need to be a type parameter of the `serialize` function? It never uses it in any way. The only type involved is `TState`.

Reply:
I guess it's not necessary here, we just need to give `SerializableReducer` a default for `action` 🤔